.cspell.json

```diff
@@ -1,6 +1,8 @@
 {
     "ignorePaths": [
         "graphics",
+        "src/assets/fonts",
+        "src/assets/styles",
```

Comment:
I don't think we should ignore spellchecking in the styles

- [ ] don't ignore spellchecking in styles

Reply:
resolved